platform_wasm/pygame/timer.py

```diff
@@ -56,15 +56,20 @@
 THREADS = {}
 
 
-def patch_set_timer(arg: Union[int, pygame.event.Event], millis: int, loops: int = 0):
-    """Patches the pygame.time.set_timer function to use gthreads"""
-
+def patch_set_timer(
+        event: Union[int, pygame.event.Event],
```

Comment:
I updated the args to match the pygame-ce api docs if that's okay with you @pmp-p  :smile: 
* https://pyga.me/docs/ref/time.html#pygame.time.set_timer
* https://pyga.me/docs/ref/event.html#pygame.event.Event.type